src/GrumPHP/EventListener/TaskEventListener.php

```diff
@@ -168,7 +195,7 @@ protected function getTaskConfigFileInfo(TaskInterface $task)
      *
      * @return array|false The configuration data or false if not supported.
      */
-    protected function readTaskConfigFile($type, $file)
+    private function readTaskConfigFile($type, $file)
```

Comment:
Waarom is private nodig?

Reply:
Class is final, no use to make it protected.